internal/config/config.go

```diff
@@ -147,10 +147,18 @@ func (caps ConfigCapabilities) Verify() []string {
 }
 
 type ConfigCapability struct {
-	AppSet             string              `yaml:"applicationset"`
-	QuotaSettings      ConfigQuotaSettings `yaml:"quotas"`
-	ExtraPermissions   ConfigCapPerm       `yaml:"extra_permissions"`
-	DefaultPermissions ConfigCapPerm       `yaml:"default_permissions"`
+	AppSet             string                       `yaml:"applicationset"`
+	QuotaSettings      ConfigQuotaSettings          `yaml:"quotas"`
+	ExtraPermissions   ConfigCapPerm                `yaml:"extra_permissions"`
+	DefaultPermissions ConfigCapPerm                `yaml:"default_permissions"`
+	CustomFields       map[string]ConfigCustomField `yaml:"custom_fields"`
+}
+
+type ConfigCustomField struct {
+	// How to validate input with a regular expression
+	Validation string `yaml:"validation"`
+	Default    string `yaml:"default"`
```

Comment:
Can you add the other descriptions as well.

Reply:
Yes, added...